### ISSUE TITLE
[CAIA A11y Research - Pilot] Practice Pilot with Dan - MHV on VA.GOV, Medical Records (March 2024)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#78985]
> [!IMPORTANT]
> - **TICKET STATUS:** `TO BE SCHEDULED`
> - **PRACTICE PILOT DATE:**  `TBD`
> - **TYPES OF ASSISTIVE TECH:** Screen Reader pilot example (mobile, desktop or both?) 
> - **PRACTICE PILOT PARTICIPANTS**: @eli-oat and @danbivins 

# CAIA A11y Support 
## Training/Practice
This is a child ticket of the research ticket [77076](https://github.com/department-of-veterans-affairs/va.gov-team/issues/77076) - created to track work, completed from the [post-pilot #78209] , practice pilot session conducted with @danbivins.

> [!TIP]
> **DEFINITION OF DONE**: Refer to the[ task list in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78985#:~:text=content%2Daccessibility%2Dia-,Definition%20of%20Done,-A11y%20Tasks), for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

-  @eli-oat 
-  @sara-amanda 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name: MHV on VA.GOV
- OCTO-DE product owner: Kaitlin Fink
- Product manager:Marci McGuire
- Designer: Lexi Wunder
- FE engineer:
- Product/team Slack channel: [mhv-medical-records](https://dsva.slack.com/archives/C03Q2UQL1AS)
- MHV a11y: Bobby Bailey and 🆕 **Dan Bivins**
- Accessibility Slack Channel for MHV: [mhv-accessibility](https://dsva.slack.com/archives/C05QG2H510T)

</details>

## Description

<details><summary>Toggle to view the description</summary>

**A CAIA a11y will participate as a user in a pilot of this research study.**

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y. `COMPLETED` See ticket - #78209 for more info.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given in the [Convo Guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/conversation-guide.md), while using and demonstrating how the behavior of a `screen reader` would behave.
- **Recording**: This session should be recorded, for additional practicing purposes.

### Pilot Session
- **Date:** `TO BE SCHEDULED`
- Prototype: [Staging Link](https://staging.va.gov/my-health/medical-records/)
- Research Convo Guide: [Research conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/conversation-guide.md)
- Research Plan: [Research plan](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/research/2024-03-usability-testing-rd2-at/research-plan.md)

</details>

### Assistive Tech Options

> [!TIP]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Review** the ["description" section in this ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/78985#:~:text=Description,view%20the%20description) (**Dan/Eli**)
- [ ] **Type of Pilot:** Mobile, Desktop or Both? (**Dan**)
- [x] **Schedule CAIA a11y:**  @danbivins to schedule with @eli-oat (**Dan**)
- [x] **Determine date and time for pilot:**  (update this ticket with date/time) (**Dan/Eli**)
- [ ] **Update CAIA A11y Capacity Calendar:** Add invite for pilot, when scheduled (**Sara**)
- [ ] **Run practice pilot** with @danbivins (**Dan/Eli**)
- [ ] **Record practice pilot** session via Zoom and share link and password in this ticket (**Eli**)
```

**Comment by danbivins** [comment]
(from the parent ticket https://github.com/department-of-veterans-affairs/va.gov-team/issues/77076) Rescheduled pilots for these dates - 3/28 (3:30pm - 5pm), 3/29 (1pm - 2:30pm and or 3pm - 4:30pm) ET zones? These will be with our AT users - Voiceover and JAWS. ET zones

**Comment by coforma-jamie** [comment]
Confirmed that this pilot happened as specified :) Closing the ticket
